e2e/windows/e2e-starter.sh

```diff
@@ -8,34 +8,34 @@ log "Starting e2e tests"
 
 # Create a resource group for the cluster
 log "Creating resource group"
-RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME-$WINDOWS_E2E_IMAGE-$K8S_VERSION"
+E2E_RESOURCE_GROUP_NAME="$E2E_RESOURCE_GROUP_NAME-$WINDOWS_E2E_IMAGE-$K8S_VERSION"
```

Comment:
I think we couldn't do that yet. Since there are two pipelines use the same yaml file (.pipelines/.vsts-vhd-builder-windows.yaml), we need to use same variable names for the reused pipeline variables.